net.c

```diff
@@ -559,8 +559,14 @@ deliver_to_host(struct qitem *it, struct mx_hostentry *host)
 	READ_REMOTE_CHECK("MAIL FROM", 2);
 
 	/* XXX send ESMTP ORCPT */
-	send_remote_command(fd, "RCPT TO:<%s>", it->addr);
-	READ_REMOTE_CHECK("RCPT TO", 2);
+	addrtmp = strdup(it->addr);
+	to_addr = strtok(addrtmp, ",");
+	while (to_addr != NULL) {
+		send_remote_command(fd, "RCPT TO:<%s>", to_addr);
+		READ_REMOTE_CHECK("RCPT TO", 2);
+		to_addr = strtok(NULL, ",");
+	}
+	free(addrtmp);
```

Comment:
I think this will leak when READ_REMOTE_CHECK fails

Reply:
Agreed, I just moved the free() to the out: which should fix the leak.